### PR TITLE
fix(docs): update Querying-data.md on proper usage of import for FilterBuilder

### PR DESCRIPTION
**Pull request by e-tang**
A update for the proper usage of import of FilterBuilder

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [x] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [x] Affected artifact templates in `packages/cli` were updated
- [x] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by dhmlau** [comment]
@e-tang, thanks for the PR. Could you please take care of the commit linter error? 
```
A update for the proper usage of import of FilterBuilder
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]
```

Based on your commit message, my suggested change would be: 
```
fix(docs): update Querying-data.md on proper usage of import for FilterBuilder
```

Please refer to the commit message guideline: https://loopback.io/doc/en/lb4/code-contrib-lb4.html#commit-message-guidelines. 

FYI - Instruction to amend commit message: https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message.

File: docs/site/Querying-data.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ const filter = {
 with the `FilterBuilder`:
 
 ```ts
-import FilterBuilder from '@loopback/repository';
+import { FilterBuilder } from '@loopback/repository';
```
Comment (raymondfeng) [inline docs/site/Querying-data.md:270]:
```ts
import {FilterBuilder} from '@loopback/repository';
```

Comment (dhmlau) [inline docs/site/Querying-data.md:270]:
```suggestion
import {FilterBuilder} from '@loopback/repository';
```

**Comment by dhmlau** [comment]
@e-tang, thanks for your contribution. Your PR has landed! 🎉

**Pull request merged**